python/shotgun_desktop/upgrade_startup.py

```diff
@@ -74,14 +76,10 @@ def upgrade_startup(splash, sgtk, app_bootstrap):
 
     try:
         latest_descriptor = current_desc.find_latest_version()
-    # This happens when a proxy blocks the connection (derives from ProxyError)
-    except httplib2.socks.HTTPError:
-        logger.warning("A proxy blocked the access to tank.shotgunstudio.com.")
-        return False
     # This happens when someone runs a local server and the Desktop doesn't have
     # internet access for security reasons.
```

Comment:
(minor) Might want to update this comment since this isn't the only case when the exception is raised. Something like

```
# Connection errors can occur for a variety of reasons. For example, there is 
# no internet access or there is a proxy server blocking access to the Toolkit app store